clap_builder/src/output/help_template.rs

```diff
@@ -478,7 +478,12 @@ impl HelpTemplate<'_, '_> {
             should_show_arg(self.use_long, arg)
         }) {
             if longest_filter(arg) {
-                longest = longest.max(display_width(&arg.to_string()));
+                let width = display_width(&arg.to_string());
+                longest = longest.max(if arg.is_positional() {
```

Comment:
I'd rather this get pulled out into a separate variable than to embed an `if` inside a parameter like this.